Packages.props

```diff
@@ -21,7 +21,7 @@
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
 		<PackageReference Update="Microsoft.SqlServer.Management.SmoMetadataProvider" Version="170.12.0" />
 		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="161.8406.0-preview" />
-		<PackageReference Update="Microsoft.SqlServer.DacFx.Projects" Version="161.8093.0-alpha" />
+		<PackageReference Update="Microsoft.SqlServer.DacFx.Projects" Version="161.8414.0-alpha" />
```

Comment:
Brings in the slash direction fix, necessary for SqlProj property tests to pass on Linux